### PR TITLE
feat(api): add routes for card info

### PR DESCRIPTION
**Pull request by ruru-m07**
image

image

### CODE REVIEW

**Comment by ruru-m07** [comment]
code cleanup is in progress.

image

**Review by ruru-m07 — COMMENTED** [review]
It looks good 👍

File: src/constants.js
Comment (ruru-m07) [inline src/constants.js:1]: i changes for me u can change for your DB name

File: src/constants.js [hunk 1]
```diff
@@ -1 +1 @@
-export const  DB_NAME = "CONCEPT"
\ No newline at end of file
+export const DB_NAME = "Universal-truth";
```
Comment (ruru-m07) [inline src/constants.js:1]: i changes for my use case u can change to your DB name

**Review by Dashrathh — APPROVED** [review]

**Pull request merged**